sql/catalyst/src/test/java/org/apache/spark/sql/catalyst/expressions/HiveHasherSuite.java

```diff
@@ -28,7 +28,7 @@
 import java.util.Set;
 
 public class HiveHasherSuite {
-  private final static HiveHasher hasher = new HiveHasher();
+  private final HiveHasher hasher = new HiveHasher();
```

Comment:
You made this non-static; that's not a style issue right? it may be OK but some of these changes don't seem to match the description